upload/src/addons/SV/WarningImprovements/Listener.php

```diff
@@ -61,4 +61,20 @@ public static function entityPreSave_XFWarningDefinition(\XF\Mvc\Entity\Entity $
 
     public static $customWarningPhrase_version_id = 101011;
     public static $customWarningPhrase_version_string = '1.1.1s';
```

Comment:
I'm not sure if setting the phrase version/id is a good idea as this is actively defeating the version tracking. And especially do not think extending the phrase entity is a good idea

The add-on actually needs to detach ownership of any user created phrases as otherwise they are nuked on upgrade. Content Ratings has some examples.

Reply:
That version id and string is only used for phrase named `warning_title.0` which has phrase_text value of "Custom warning" and whenever a warning is edited MasterTitle method is under XF:Phrase entity to update (even if there are no changes).